doc/classes/CollisionObject3D.xml

```diff
@@ -250,17 +251,19 @@
 			<argument index="4" name="shape_idx" type="int">
 			</argument>
 			<description>
-				Emitted when [method _input_event] receives an event. See its description for details.
+				Emitted when [method _input_event] receives an event. Requires [member input_ray_pickable] to be [code]true[/code] and at least one [member collision_layer] bit to be set. See [method _input_event] for details.
 			</description>
 		</signal>
 		<signal name="mouse_entered">
 			<description>
-				Emitted when the mouse pointer enters any of this object's shapes.
+				Emitted when the mouse pointer enters any of this object's shapes. Requires [member input_ray_pickable] to be [code]true[/code] and at least one [member collision_layer] bit to be set.
+				[b]Note:[/b] Due to the lack of continuous collision detection, this signal may not be emitted if the mouse moves fast enough and the [CollisionObject2D]'s area is small. This signal may also not be emitted if another [CollisionObject2D] is overlapping the [CollisionObject2D] in question.
```

Comment:
Based on https://github.com/godotengine/godot/issues/49286#issuecomment-854912653, it looks like no event is actually skipped, but they might be received in a different order than expected when the mouse moves fast.

Reply:
I reworded the note to follow your suggestion. I haven't verified the behavior myself though.